projects/packages/forms/src/contact-form/js/accessible-form.js

```diff
@@ -775,9 +844,23 @@ const setFormError = form => {
 		}
 	}
 
-	error.appendChild(
-		createError( L10N.invalidForm || 'Please make sure all fields are correct.' )
+	const { disableLiveRegion } = opts;
+
+	if ( disableLiveRegion ) {
```

Comment:
We want screen readers to announce the message upon submitting the form but not when a user fixes a field.